src/app/manager-dashboard/certifications/certifications-view.component.html

```diff
@@ -6,16 +6,26 @@
 </mat-toolbar>
 
 <div class="space-container">
-  <mat-toolbar class="primary-color font-size-1">
-    {{certification.name}}
+  <mat-toolbar class="primary-color font-size-1" >
+   {{certification.name}}
+   <span class="empty-spacer"></span>
+   <button mat-stroked-button (click)="routeToEdit(certification?._id)">
+    <mat-icon>edit</mat-icon>Edit
+   </button>
   </mat-toolbar>
   <div class="view-container view-full-height">
     <mat-tab-group>
       <mat-tab i18n-label label="Course Requirements">
-        <planet-courses *ngIf="certification.courseIds.length > 0" [displayedColumns] = "[ 'courseTitle', 'info', 'createdDate', 'rating' ]" [isForm]="true" [includeIds]="certification.courseIds"></planet-courses>
+        <planet-courses *ngIf="certification.courseIds.length > 0; else noCoursesMessage" [displayedColumns] = "[ 'courseTitle', 'info', 'createdDate', 'rating' ]" [isForm]="true" [includeIds]="certification.courseIds"></planet-courses>
+        <ng-template #noCoursesMessage>
+          <div class="view-container">No courses has been added. Click on <b>Edit</b> to add courses.</div>
```

Comment:
Three things to add here and line 27:

1. You don't need `class="view-container"` here
2. You should add an `i18n` tag so the text will be translated
3. It should be `have` instead of `has`